### ISSUE TITLE
Synchronous getSwatches & browserify

### ISSUE THREAD

**Issue by JesseFarebro** [akfish/node-vibrant#2]
I've been trying to use Vibrant in an isomorphic setting and the asynchronous getSwatches doesn't match the API of Vibrant directly.

As well adding a browser field to package.json and add vibrant.js as a dependency because this is the only published vibrant package on npm.

**Comment by akfish** [comment]
node-vibrant does not run in browser environment, yet. 
I deviated from vibrant.js's API on purpose for the following reasons:
* Synchronous API is neither user-friendly nor developer-friendly. It blocks UI thread and make batch processing awkward.
* Most node.js image processing package runs asynchronously. 
* Original vibrant.js API does not feel right in node.js's programming paradigm. For example, you cannot use Promise with constructor.

Fortunately we have web worker in browser environment. So instead I plan to bring asynchronous API to browser, not the other way around. Hopefully that will be good enough for you.

**Comment by akfish** [comment]
Browser & browserify support added in v1.1.0. Identical API is used for both browser and node.js environments. 

As mentioned in last comment, I implemented the asynchronous API for browser, except that I dropped the web worker approach since it will introduce un-necessary complexity and the performance is not that bad anyway. Above all, the original vibrant.js workflow is asynchronous in essence (you will have to load and handle `image.onload` event by yourself.)

I hope this will solve your problem. :)
